pkg/virtctl/create/instancetype/instancetype.go

```diff
@@ -59,16 +61,6 @@ type createInstancetype struct {
 	namespaced      bool
 }
 
-type gpu struct {
-	Name       string `param:"name"`
-	DeviceName string `param:"devicename"`
-}
```

Comment:
Can you keep a single struct here instead of defining it inline in `processDeviceParams`?